### PR TITLE
Add X Display support (simple method)

### LINKED ISSUE #44: Add display support
Though the docker-compose file shares the X11 unix socket:

https://github.com/orise-robotics/ros_ws/blob/cf7326fca27913a8b78814685555831aff5219b9/docker-compose.yml#L55

The package does not offer resources to make graphical applications work from within the container.

This [Tutorial](http://wiki.ros.org/docker/Tutorials/GUI) is a good start point to solve this problem. I suggest going with the simplest method presented there even though it has security issues, then map another issue for optimization.

### PR DESCRIPTION
**Pull request by mateus-amarante**
### Description

Implement X Display support. It allows the user to run graphic applications inside the container. It implements the simplest method presented in [this](http://wiki.ros.org/docker/Tutorials/GUI) tutorial

Vide #44

### Related Issues:
- Fix #44

### CODE REVIEW

**Review by pxalcantara — APPROVED** [review]

File: run-devel.sh [hunk 1]
```diff
@@ -26,6 +27,9 @@ while [ -n "$1" ]; do
     ROS_DISTRO=$2
     shift
     ;;
+  -x | --display)
```
Comment (ivan1993br) [inline run-devel.sh:30]: Why just sharing the `DISLAY` environment variable is not enough?

Comment (mateus-amarante) [reply run-devel.sh:30]: It must have access and permission to the x server socket. Generally, I followed "the simple way" described in [this](http://wiki.ros.org/docker/Tutorials/GUI) tutorial

Comment (ivan1993br) [reply run-devel.sh:30]: But why specify the `XDISPLAY` vairable? I mean the socket we share at its default locaution `/tmp/.X11-unix` and the display we say which is with `DISPLAY` env. What is the error we get if we do not do this step?

Comment (mateus-amarante) [reply run-devel.sh:30]: You can check by running `gzserver --verbose`. Without the `-x` I get:

`[Err] [RenderEngine.cc:749] Can't open display: :1`

**Review by ivan1993br — APPROVED** [review]

**Pull request merged**